core/unix/rseq_linux.c

```diff
@@ -768,6 +865,9 @@ rseq_locate_rseq_regions(void)
 void
 rseq_module_init(module_area_t *ma, bool at_map)
 {
+    /* If !rseq_enabled, the module will simply be processed when we enable
+     * rseq.
```

Comment:
I would have expected the glibc checking code to be inside rseq_process_module, instead of having it doing its own module walk from multiple places in addition to the module walk we already have in rseq_locate_rseq_regions() -- should we move the rseq_check_glibc_support() to not have its own loop and be called from rseq_process_module?

Reply:
Removed.